generate/generate.go

```diff
@@ -764,75 +772,35 @@ func (g *Generator) SetupPrivileged(privileged bool) {
 			finalCapList = append(finalCapList, fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())))
 		}
 		g.initSpecLinux()
-		g.spec.Process.Capabilities = finalCapList
+		g.spec.Process.Capabilities.Bounding = finalCapList
+		g.spec.Process.Capabilities.Effective = finalCapList
+		g.spec.Process.Capabilities.Inheritable = finalCapList
+		g.spec.Process.Capabilities.Permitted = finalCapList
+		g.spec.Process.Capabilities.Ambient = finalCapList
 		g.spec.Process.SelinuxLabel = ""
 		g.spec.Process.ApparmorProfile = ""
 		g.spec.Linux.Seccomp = nil
 	}
 }
 
-// ClearProcessCapabilities clear g.spec.Process.Capabilities.
-func (g *Generator) ClearProcessCapabilities() {
```

Comment:
Clearing *all* of `process.capabilities` is still a well-defined operation.  We may need *new* operations to clear individual `process.capabilities.*`, but there's no need to drop this one.